contrib/capitalone_dataprofiler_expectations/capitalone_dataprofiler_expectations/tests/conftest.py

```diff
@@ -127,11 +127,6 @@ def pytest_addoption(parser):
         action="store_true",
         help="If set, execute tests against snowflake",
     )
-    parser.addoption(
-        "--aws-integration",
-        action="store_true",
-        help="If set, run aws integration tests for usage_statistics",
```

Comment:
This is really all this flag gates? It's such a general time for a usage stats test.

Reply:
yup a quick grep shows that its no longer used (and was only used for usage stats before)